swifttool/client.py

```diff
@@ -189,6 +244,14 @@ def generate_script(self, outdir, name='ring_builder.sh', rebalance=True,
         os.chmod(outfile, st.st_mode | stat.S_IEXEC)
         return outfile
 
+    def is_devpresent(self, ringtype, zone, node, port, disk):
+        command = self._ring_search_command(ringtype, zone, node,
+                                            port, disk)
+        with open(os.devnull, 'w') as devnull:
+            rc = subprocess.call(command, shell=True, stdout=devnull,
```

Comment:
@j2sol Ring search outputs device info on stdout. In past we had run into issue when subprocess would randomly terminate with stdout buffer overflows. That probably wont be case here though... Do you want me to remove the devnull?